sklearn/datasets/descr/california_housing.rst

```diff
@@ -0,0 +1,36 @@
+
+.. _california_housing:
+
+The California Housing dataset
+================================
+
+
+This data set contains a set of housing information taken 
+in the 1990 Census (in the US). The :func:`sklearn.datasets.fetch_california_housing`
+function is the data fetching / caching function that downloads the data
+archive from AT&T.
+
+.. _This data set from the original source:

```

Comment:
I'm not actually sure what you intend here. THis formatting doesn't display the words. See https://18174-843222-gh.circle-artifacts.com/0/doc/datasets/index.html#the-california-housing-dataset.

Just say "The dataset is originally from "